docs/source/markdown/podman-container-diff.1.md

```diff
@@ -1,54 +0,0 @@
-% podman-container-diff 1
```

Comment:
This should not be removed. `podman diff` is not the same. It can accept either an image or container as argument while `podman container diff` only support containers.

Reply:
Readded.